rules/ng_file_name.js

```diff
@@ -0,0 +1,96 @@
+module.exports = (function () {
+    'use strict';
+
+    var utils = require('./utils/utils');
+    var path = require('path');
+    var fileEnding = '.js';
+
+    var separators = {
+        dot: '.',
+        dash: '-',
+        underscore: '_'
+    };
+
+    var componentTypeMappings = {
+        module: 'module',
+        controller: 'controller',
```

Comment:
Is it the most used suffix for NG component ? If yes, for controller, you can add "ctrl" also maybe.